### PR TITLE
Add stat var count in hierarchy UI

### PR DESCRIPTION
**Pull request by shifucun**
* Show descendent stat var count as super script 
* Gray out nodes that have no descendent stat vars
* Disable input when stat var has no data.

image

image

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
nice!

File: static/css/tools/stat_var_widget.scss [hunk 1]
```diff
@@ -97,3 +97,7 @@ $transparent-red: rgba(240, 230, 231, 0.2);
   cursor: pointer;
   background-color: $transparent-gray;
 }
+
+#hierarchy-section .node-no-data {
```
Comment (chejennifer) [inline static/css/tools/stat_var_widget.scss:101]: looks like .node-no-data with opacity 30% is also set in stat_var_hierarchy.tsx. Does it need to be set in both places?

**Review by chejennifer — APPROVED** [review]

File: static/css/tools/stat_var_widget.scss (hunk 1 above)
Comment (shifucun) [reply static/css/tools/stat_var_widget.scss]: Removed it here.

**Review by beets — COMMENTED** [review]
i think it's confusing having both numbers there. some options below. 

image
image
image

it's unfortunate that we use red so much, since it could be construed as an error .. here is a try with green.
image

File: static/js/stat_var_hierarchy/node_header.tsx [hunk 2]
```diff
@@ -29,7 +29,8 @@ const DOWN_ARROW_HTML = <i className="material-icons">remove</i>;
 const RIGHT_ARROW_HTML = <i className="material-icons">add</i>;
 
 interface StatVarHierarchyNodeHeaderPropType {
-  count: number;
+  statVarCount: number;
```
[the remark this answers is not in the document]
Comment (shifucun) [reply static/js/stat_var_hierarchy/node_header.tsx]: Done

**Review by shifucun — COMMENTED** [review]
> i think it's confusing having both numbers there. some options below.
> 
> image
> image
> image
> 
> it's unfortunate that we use red so much, since it could be construed as an error .. here is a try with green.
> image

Use highlighted blue now.

File: static/css/stat_var_hierarchy.scss [hunk 3]
```diff
@@ -106,6 +106,10 @@ $horizontal-spacing: 15px;
   background-color: var(--dc-red-fade);
 }
 
+.selected-node-title {
+  color: #67adff;
```
Comment (beets) [inline static/css/stat_var_hierarchy.scss:110]: make the color a var?

**Review by beets — APPROVED** [review]

File: static/css/stat_var_hierarchy.scss (hunk 3 above)
Comment (shifucun) [reply static/css/stat_var_hierarchy.scss]: Done

**Pull request merged**